Resources/Prototypes/Reagents/aphrodisiac.yml

```
@@ -0,0 +1,16 @@
+- type: reagent
+  id: Aphrodisiac
+  name: reagent-name-aphrodisiac
+  group: Narcotics
+  desc: reagent-desc-aphrodisiac
+  physicalDesc: reagent-physical-desc-powdery
+  flavor: bitter
+  color: "#63806e"
+  metabolisms:
+    Aphrodisiac:
```

```suggestion
    Poison:
```
Очень грубая ошибка в коде